### DISCUSSION TITLE
Make a subdomain that forwards to member onboarding page

### DISCUSSION THREAD

**Discussion by bmann** [memser-spaceport/pl-network-hub#11] [Ideas]
I just got an updated link to the member onboarding form at https://airtable.com/shr7xzCLmEiatgcNK

Suggestion: make a subdomain like `memberonboarding.plnetwork.io` and either embed the form or forward to the Airtable link. 

That way PLN members can embed it in their onboarding and not have to change it, and Spaceport can point it at whatever page or service is needed over time.

**Comment by laeccles** [comment]
@pandeyanujk 

Denice shared that she would like this to live in plnetwork.io, possibly on the homepage.

cc:  @DeniceZ

**Reply by pandeyanujk** [reply]
@bmann @laeccles noted. The issue here is really the link rot or bad link when things/tools are changed. Putting the onboarding flow into the portal (plnetwork.io) is a good call-out. We will add this request to the backlog & groom it into an upcoming sprint. I will send you an update on the progress. Thanks!
